[2023-08-28T00:27:59.037+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2 permission to 509
[2023-08-28T00:27:59.039+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2'
[2023-08-28T00:27:59.086+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2 permission to 509
[2023-08-28T00:27:59.087+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2'
[2023-08-28T00:27:59.253+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_bash_with_macro_eg2.bash_task_t2 manual__2023-08-28T00:27:57.109448+00:00 [queued]>
[2023-08-28T00:27:59.261+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_bash_with_macro_eg2.bash_task_t2 manual__2023-08-28T00:27:57.109448+00:00 [queued]>
[2023-08-28T00:27:59.261+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-28T00:27:59.296+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): bash_task_t2> on 2023-08-28 00:27:57.109448+00:00
[2023-08-28T00:27:59.301+0000] {standard_task_runner.py:57} INFO - Started process 295 to run task
[2023-08-28T00:27:59.305+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_bash_with_macro_eg2', 'bash_task_t2', 'manual__2023-08-28T00:27:57.109448+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/dags_bash_with_macro_eg2.py', '--cfg-path', '/tmp/tmp50wm64wy']
[2023-08-28T00:27:59.308+0000] {standard_task_runner.py:85} INFO - Job 78: Subtask bash_task_t2
[2023-08-28T00:27:59.385+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2 permission to 509
[2023-08-28T00:27:59.386+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_with_macro_eg2/run_id=manual__2023-08-28T00:27:57.109448+00:00/task_id=bash_task_t2'
[2023-08-28T00:27:59.389+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_bash_with_macro_eg2.bash_task_t2 manual__2023-08-28T00:27:57.109448+00:00 [running]> on host 07fcf706bef2
[2023-08-28T00:27:59.519+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_bash_with_macro_eg2' AIRFLOW_CTX_TASK_ID='bash_task_t2' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:27:57.109448+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-28T00:27:57.109448+00:00'
[2023-08-28T00:27:59.523+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-28T00:27:59.524+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "START_DATE: $START_DATE" && echo "END_DATE: $END_DATE"']
[2023-08-28T00:27:59.535+0000] {subprocess.py:86} INFO - Output:
[2023-08-28T00:27:59.536+0000] {subprocess.py:93} INFO - START_DATE: 2023-07-10
[2023-08-28T00:27:59.537+0000] {subprocess.py:93} INFO - END_DATE: 2023-07-15
[2023-08-28T00:27:59.537+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-28T00:27:59.592+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dags_bash_with_macro_eg2, task_id=bash_task_t2, execution_date=20230828T002757, start_date=20230828T002759, end_date=20230828T002759
[2023-08-28T00:27:59.638+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-28T00:27:59.654+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
